writers/streamingbatchwriter/streamingbatchwriter.go

```diff
@@ -158,9 +158,9 @@ func (w *StreamingBatchWriter) stopWorkers() {
 	}
 	w.workersWaitGroup.Wait()
 
-	w.insertWorkers = make(map[string]*streamingWorkerManager[*message.WriteInsert])
-	w.migrateWorker = nil
-	w.deleteWorker = nil
```

Comment:
Not setting both of these to `nil` means if there's a write of their type, the channel write will panic with 'write on closed channel'